### PR TITLE
feat: add unblockmessage

### PR DESCRIPTION
**Pull request by ffenix113**
## Changelog

- Add `_unblockMessage` private function

Chat MR: https://github.com/GetStream/chat/pull/2969

### CODE REVIEW

File: src/client.ts [hunk 1]
```diff
@@ -2303,6 +2303,24 @@ export class StreamChat<
     });
   }
 
+  /**
+   * unblockMessage - unblocks message blocked by automod
+   *
+   * Note: Do not use this.
+   * It is present for internal usage only.
+   * This function can, and will, break and/or be removed at any point in time.
+   *
```
Comment (mahboubii) [inline src/client.ts:2312]:
```suggestion
   *
   * @private
```

**Review by mahboubii — DISMISSED** [review]

File: src/client.ts (hunk 1 above)
Comment (ffenix113) [reply src/client.ts:2312]: Thanks for this, will add for other new private methods.

Comment (ffenix113) [reply src/client.ts:2314]: Updated, please check.

**Review by mahboubii — APPROVED** [review]

**Review by ferhatelmas — APPROVED** [review]

**Comment by ffenix113** [comment]
@peterdeme Seems like `${{ github.event.pull_request.title }}` does not contain latest changes, only initial event, so it will not pass PR name lint if firstly PR name was incorrect. Can we take it into account please?

**Review by peterdeme — APPROVED** [review]

**Pull request merged**